extras/container/heketi-start.sh

```diff
@@ -70,6 +70,32 @@ if [[ $flock_status -ne 0 ]]; then
     fail "Database file is read-only"
 fi
 
+# this creates archival copies of the db if needed for disaster
```

Comment:
Isn't it better to to this after line 114? That would ensure the archival works with secret based db storage.

Reply:
I wan't sure if that's actually the behavior we wanted. That next block overwrites whatever was the previous "database file" with the externally backed up copy. If we move this block down we lose the ability to know what was there prior to that restore. Since we have no control over what's in that location I was worried we might be losing interesting state if we copied the backup into the archive rather than the current contents of heketi.db.

Reply:
Ok, that is a valid point. I am happy with archival being enabled on for users who don't store on secret.